src/retrievers/getStores.js

```diff
@@ -0,0 +1,46 @@
+const objectAssign = require('object-assign')
+const Store = require('../models/Store')
+const Pagination = require('../models/Pagination')
+const request = require('../util/request')
+
+const defaults = {page: 1}
+
+function getStores(opts) {
+  const options = objectAssign({}, defaults, opts || {})
+  const query = {
+    q: '*',
+    page: Math.max(0, options.page - 1)
+  }
+
+  if (options.lat && options.lon) {
```

Comment:
I expect this is the center point of a "nearby"-type search? Perhaps we could use a `near` or `nearLocation` key to make this even more obvious?

Reply:
Yeah, does that feel right to you?

Reply:
Sounds good to me.

Reply:
(fixed by the way)